webapp/src/main/resources/templates/index.html

```diff
@@ -7,6 +7,7 @@
         <script type="text/javascript">
             APP_CONFIG = {{{appConfig}}};
             CSRF_TOKEN = '{{csrfToken}}';
+            USER_MENU_LOGOUT_HIDDEN = {{userMenuLogoutHidden}};
```

Comment:
Could we not add this to the app config? 
I don't want this page to get bloated with individual variables every time there is a new UI level parameter added

Reply:
@byronantak - could you also share some thoughts on how an alternative could be implemented better, without having an app config setting for this?

Reply:
Done